bin/local-env/install-composer.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+COMPOSER_VERSION=`curl -Ls -w %{url_effective} -o /dev/null https://github.com/composer/composer/releases/latest | rev | cut -d '/' -f 1 | rev`
+
+# Exit if any command fails
+set -e
+
+# Include useful functions
+. "$(dirname "$0")/includes.sh"
+
+
+# Change to the expected directory
+cd "$(dirname "$0")/../.."
+
+# Check if composer is installed
+if [ "$TRAVIS" != "true" ] && ! command_exists "composer"; then
+	if ask "$(error_message "Composer isn't installed, would you like to download and install it automatically?")" Y; then
+		echo -en $(status_message "Installing Composer..." )
+		download "https://getcomposer.org/installer" | bash >/dev/null 2>&1
+		mkdir -p /usr/local/bin
+		mv composer.phar /usr/local/bin/composer
+		echo ' done!'
+	else
+		echo -e $(error_message "")
+		echo -e $(error_message "Please install Composer manually, then re-run the setup script to continue.")
+		echo -e $(error_message "Composer installation instructions can be found here: $(action_format "https://getcomposer.org/doc/00-intro.md")")
+	fi
+
+	exit 1
+fi
+
+# Check if the current Composer version is up to date.
+if [ "$TRAVIS" != "true" ] && ! [[ "$(composer --version)" == "Composer version $COMPOSER_VERSION "* ]]; then
+	echo -en $(status_message "Updating Composer..." )
+	composer self-update
+	echo ' done!'
+fi
+
+
+# Install/update packages
+echo -e $(status_message "Installing and updating Composer packages..." )
+composer install
```

Comment:
```suggestion
composer install --optimize-autoloader --no-interaction --prefer-dist --no-suggest --ignore-platform-reqs
```

Some tweak to composer install. 

- --optimize-autoloader Optimizes the autoloader and make vendor auto loaded libraries a little faster as it uses a class map. 
- --no-interaction  Do not ask questions. 
- --prefer-dist Get dist versions instead of git checkouts. Builds faster. 
- --no-suggest Don't both to suggest other libraries. 
-  --ignore-platform-reqs Ignore currently platform and just build.

Reply:
Done in 17d85592f347c619342437f5c1061221b5b1aed9